### PR TITLE
add script for tests on IBMC

### PR DESCRIPTION
**Pull request by anandrkskd**
Signed-off-by: anandrkskd <anandrkskd@gmail.com>

**What type of PR is this?**

 /kind feature

**What does this PR do / why we need it**:

This PR adds script to run tests on Ibm cloud 

**Which issue(s) this PR fixes**:

Fixes - part of #4412
Fixes - part of #4408

**PR acceptance criteria**:

- [ ] Unit test 

- [ ] Integration test 

- [ ] Documentation 

- [ ] Update changelog

- [x] I have read the [test guidelines](https://github.com/openshift/odo/blob/master/docs/dev/test-architecture.adoc)

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

**Review by rnapoles-rh — COMMENTED** [review]
LGTM

File: scripts/run_script_e2e.sh [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+# Prep for integration/e2e
+shout "Building odo binaries"
+make bin
+
+# copy built odo to GOBIN
+cp -avf ./odo $GOBIN_TEMP/
+shout "getting ginkgo"
+make goget-ginkgo
```
[the remark this answers is not in the document]
Comment (anandrkskd) [reply scripts/run_script_e2e.sh:16]: Actually we need to do this because we are using different $GOBIN for each build so we will need this.

**Pull request closed without merging**